index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Urban Fictionary</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Shopping Cart Page">
+    <meta name="author" content="Rachel">
+    <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/custom.css" rel="stylesheet">
+  </head>
+  <body>
+<!-- Landing Page -->
+    <h1>Urban Fictionary</h1>
+      <section id="page">
+        <form>
+          <label>Player 1: <input type="text" placeholder="Name" /></label>
+          <button type="button" class="btn">Submit</button><br />     
+          <label>Player 2: <input type="text" placeholder="Name" /></label>
+          <button type="button" class="btn">Submit</button><br />    
+        </form>
+        <p class="introduction">
+          Description: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+        </p>
+        <button type="button" class="btn btn-lg">Start the Game!</button>
+       </section>   
+
+
+<!-- Game Page  -->
+    <div class="container" id="page">
+      <div class="row">
+  <!-- Player Details     -->
+        <div class="col-xs-3">
+
+        </div>
+  <!-- Main Game       -->
+        <div class="col-xs-6 accordion" id="Q1">
+          <h3> Question 1 </h3>
+            <div class="pane">
+              <h4>Just Ducky</h4>
+              <ul>
+                <li>Choice A</li>
+                <li>Choice B</li>
+                <li>Choice C</li>
+              </ul>
+            </div>
+          <h3> Question 2 </h3>
```

Comment:
Further reference: You should generate / append questions dynamically, instead of `hard coding` them